storage/bootstrap.go

```diff
@@ -36,16 +36,14 @@ import (
 
 type bootstrapState int
 
+// nolint: deadcode
```

Comment:
We should also potentially add a `GUIDELINE.md` for resolving common lint failures in the `ci-scripts` repo so that users know what their options are when they receive a certain type of lint error.